index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>サイト制作課題</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Anton&family=Noto+Sans+JP:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header class="header">
+      <h1 class="header-logo-wrapper">
+        <img class="header-logo" src="./img/logo.svg" alt="Grand Players" />
+      </h1>
+    </header>
+  </body>
+  <section class="hero">
+    <div class="hero-container">
+      <div class="hero-texts">
+        <p class="hero-main-copy">
+          BE<br />A<br />
+          <span class="hero-main-copy-accent">"GRAND"</span>
```

Comment:
細かいですがダブルクォーテーションマークは全角になります。
```suggestion
          <span class="hero-main-copy-accent">“GRAND”</span>
```